### PR TITLE
Sourcerank

### LINKED ISSUE #8088: ENH: Improve our sourcerank score
See some entries missing in

https://libraries.io/pypi/mne/sourcerank

compared to 

https://libraries.io/pypi/nibabel/sourcerank

For 0.21 we should fix the easy missing ones. Something trivial for someone to do as a Sunday hack...

### PR DESCRIPTION
**Pull request by drammock**
closes #8088

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -72,6 +72,12 @@ def package_tree(pkgroot):
                        'Operating System :: MacOS',
                        'Programming Language :: Python :: 3',
                        ],
+          keywords='neuroscience neuroimaging MEG EEG ECog fNIRS brain',
```
Comment (larsoner) [inline setup.py:75]:
```suggestion
          keywords='neuroscience neuroimaging MEG EEG ECoG fNIRS brain',
```

**Comment by larsoner** [comment]
Any need to iterate further? I guess the JSON is created by PyPi rather than being part of the upload directly. So +1 for merging and backporting this and seeing if eventually 0.21.1 looks better

**Comment by larsoner** [comment]
... or you could upload something to test.pypi.org and look at

https://test.pypi.org/pypi/mne/json

**Comment by drammock** [comment]
OK, just created account `drammock` on test.pypi.org

I'm assuming you need to grant me access to `mne` there?

**Comment by larsoner** [comment]
I'm not an owner there, so it'll have to be @agramfort

**Comment by drammock** [comment]
Looks like it worked!  the new URLs are in [the JSON file](https://test.pypi.org/pypi/mne/0.22.dev0/json) and the site populates them into the sidebar:

Screenshot_2020-09-23_14-35-32

Merging.

**Comment by drammock** [comment]
backported

**Comment by larsoner** [comment]
Awesome, thanks @drammock

**Pull request merged**